Backpack/src/main/res/values-large/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <!-- internal resources -->
```

Comment:
What makes this internal?

Reply:
It's just a comment, we have the same in other dimens files. Making resources private is another task.